Quantum-Control-Applications/ESR/configuration.py

```diff
@@ -10,74 +15,71 @@ def IQ_imbalance(g, phi):
     return [float(N * x) for x in [(1 - g) * c, (1 + g) * s, (1 - g) * s, (1 + g) * c]]
 
 
-def gaussian_rise(amp, pulse_len, flat_top, sigma):
-    gauss_length = int((pulse_len - flat_top) * 2)
-    gauss = (amp * gaussian(gauss_length, sigma)).tolist()
-    return gauss[: gauss_length // 2] + [amp] * flat_top
-
-
 #############
 # VARIABLES #
 #############
-
-qop_ip = "127.0.0.1"
-qop_port = 80
+# API allowing the use of unit and data conversion
+u = unit()
+# IP address of the Quantum Orchestration Platform
+qop_ip = "172.16.2.103"
```

Comment:
wait, isn't one of our default values "192.168.88.10" or maybe some other but why to leave the one from the office?

Reply:
I agree.
We usually do `127.0.0.1` (which is a default for your own computer) as a placeholder.

Reply:
Yes correct I always forget to put it back sorry...